tests/istio.mk

```diff
@@ -215,6 +215,14 @@ helm/init: ${HELM}
 
 # Install istio for the first time
 helm/install:
+	${HELM} install \
+	  install/kubernetes/helm/istio-init \
+	  --name istio-system-init --namespace istio-system \
+	  --set global.hub=${HUB} \
```

Comment:
is hub or tag var relevant?  I thought you said earlier no config nob for this chart.

Reply:
yes, there will be config knobs, as a container needs to be run to convert configmaps to CRDs.  I will modify the docs in a followon.